src/Controls/src/Core/HandlerImpl/Page/Page.Android.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System;
+namespace Microsoft.Maui.Controls;
+
+public partial class Page
```

Comment:
If we're going to create all these new files can we just make a small push to reorganize these page files like I have on this PR?

https://github.com/dotnet/maui/pull/14339

Reply:
@PureWeen let me know if this looks good now!